sources/models/string_impls_for/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "string_impls_for"
+version = "0.1.0"
+authors = ["Sean P. Kelly <seankell@amazon.com>"]
```

Comment:
Let's add the original authors here too since it's refactoring out of an existing crate.